packages/modules/imodel-browser/src/containers/iModelGrid/useIModelData.ts

```diff
@@ -126,13 +131,13 @@ export const useIModelData = ({
     };
   }, [
     accessToken,
-    projectId,
-    apiOverrides?.data,
```

Comment:
Why was the `apiOverrides.data` removed from that dependency list ? If we only change the data in the existing object, we'd expect the data to be updated (if this really caused a problem, we could decide that the whole `apiOverrides` object needs to change, but if we can, let's keep the previous behavior)

Reply:
I have added `apiOverrides.data`